main.py

```diff
@@ -58,10 +54,8 @@
 
 
 def browser_setup():
-    user_agent = 'Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:109.0) Gecko/20100101 Firefox/109.0'
     firefox_option = Options()
-    firefox_option.add_argument('-headless')
-    firefox_option.set_preference('general.useragent.override', user_agent)
+    firefox_option.add_argument('--headless')
```

Comment:
headless flag broken on my instance, needed the `--` to work. Also remove the user_agent as it didn't seem needed